README.md

```diff
@@ -161,39 +150,19 @@ module "cdn" {
 
 - [Complete](https://github.com/terraform-aws-modules/terraform-aws-cloudfront/tree/master/examples/complete) - Complete example which creates AWS CloudFront distribution and integrates it with other [terraform-aws-modules](https://github.com/terraform-aws-modules) to create additional resources: S3 buckets, Lambda Functions, CloudFront Functions, VPC Origins, ACM Certificate, Route53 Records.
 
-## Notes
```

Comment:
This is now solved by these added guards (both default and ordered cache behaviors) so users don't need to worry about this https://github.com/terraform-aws-modules/terraform-aws-cloudfront/pull/177/files#diff-dc46acf24afd63ef8c556b77c126ccc6e578bc87e3aa09a931f33d9bf2532fbbR36-R37